### DISCUSSION TITLE
Cannot Install Chatwoot on Oracle Cloud

### DISCUSSION THREAD

**Discussion by DhanwanthParameswar** [chatwoot/chatwoot#5544] [Self hosted]
Hey guys! I am currently trying to run chatwoot on an oracle cloud server with 2 Ampere A1 Flex CPUs, 12 GB Memory, and 100 GB disk space. I am using cloud flare to manage my dns. I want to install this on chat.casmat.ga . I have pointed the host of chat.casmat.ga in cloudflare using an A record to my instance's ip address: 129.213.192.48 I used the installation guide, but when opening the chat.casmat.ga site, it shows cloudflare error. I also am wondering on how to remove the powered by chatwoot branding for the open source installation. I'm new to all this, so any help would be useful! Thanks again!

**Comment by pranavrajs** [comment]
Moving this to a discussion as it doesn't seem to be an issue with the Chatwoot instance. Can you please share any error logs or screenshots from the Cloudflare account which can help debug this?
